shared_libs/transformer_base_lib/transformer_base_lib/app_config.py

```diff
@@ -30,7 +30,7 @@ def read_env_config() -> AppConfig:
             egress_queue_name=_read_env("EGRESS_QUEUE_NAME", required=True),
             egress_session_id=_read_env("EGRESS_SESSION_ID", required=False),
             service_bus_namespace=_read_env("SERVICE_BUS_NAMESPACE", required=False),
-            audit_queue_name=_read_env("AUDIT_QUEUE_NAME", required=True),
+            audit_queue_name=_read_env("AUDIT_QUEUE_NAME", required=False),
```

Comment:
not required on PHW, but set as required on the other apps - which one should it be?

Reply:
okay leaving as is